coreos-ostree-importer/Dockerfile

```diff
@@ -12,8 +12,8 @@ RUN dnf update -y
 RUN dnf -y install \
```

Comment:
Can drop the comment just before this line now.

Reply:
done